base/Base.jl

```diff
@@ -139,7 +139,7 @@ include("abstractdict.jl")
 
 include("iterators.jl")
 using .Iterators: zip, enumerate
-using .Iterators: Flatten, Filter, product  # for generators
+using .Iterators: Flatten, Filter, product # for generators
```

Comment:
Why?

Reply:
leaves whitespace after code alone for these kind of comments now